spec/dfe/analytics/services/entity_table_checks_spec.rb

```diff
@@ -294,14 +296,13 @@
       end
     end
 
-    it 'orders records by updated_at truncated to milliseconds' do
-      time_base = Time.zone.now.beginning_of_minute
-      Candidate.create(email_address: 'first@example.com', updated_at: time_base + 0.001.seconds)
-      Candidate.create(email_address: 'second@example.com', updated_at: time_base + 0.005.seconds)
+    it 'orders records by created_at truncated to milliseconds' do
+      Candidate.create(email_address: 'first@example.com')
+      Candidate.create(email_address: 'second@example.com')
 
       described_class.call(entity_name: candidate_entity, entity_type: entity_type, entity_tag: nil)
 
-      ordered_candidates = Candidate.order(:updated_at).pluck(:email_address)
+      ordered_candidates = Candidate.order(:created_at).pluck(:email_address)
```

Comment:
This test may be flaky at times if the 2 records are created in the same millisecond. Not sure how you can fix this if passing in `created_at` is ignored.